src/libraries/Common/src/Interop/Windows/WinSock/Interop.WSAConnect.cs

```diff
@@ -12,7 +12,7 @@ internal static partial class Winsock
         [LibraryImport(Interop.Libraries.Ws2_32, SetLastError = true)]
         internal static partial SocketError WSAConnect(
             SafeSocketHandle socketHandle,
-            byte[] socketAddress,
+            Span<byte> socketAddress,
```

Comment:
Should we make the actual LibraryImport private, with an internal wrapper that doesn't take `socketAddressSize`? e.g.
```
[LibraryImport(Interop.Libraries.Ws2_32, SetLastError = true)]
private static partial SocketError WSAConnect(
    SafeSocketHandle socketHandle,
    Span<byte> socketAddress,
    int socketAddressSize,
    IntPtr inBuffer,
    IntPtr outBuffer,
    IntPtr sQOS,
    IntPtr gQOS);

internal static SocketError WSAConnect(
    SafeSocketHandle socketHandle,
    Span<byte> socketAddress,
    IntPtr inBuffer,
    IntPtr outBuffer,
    IntPtr sQOS,
    IntPtr gQOS) =>
    WSAConnect(socketHandle, socketAddress, socketAddress.Length, inBuffer, outBuffer, sQOS, gQOS);
```
?